packages/wallet-service/src/db/index.ts

```diff
@@ -2389,7 +2390,7 @@ export const filterTxOutputs = async (
     ignoreLocked: false,
     skipSpent: true,
     biggerThan: -1,
```

Comment:
You changed the default in the other file in the Joi schema to `0n`. I think it doesn't affect anything but it would be good to have both the same.

Reply:
Changed here as well